index.js

```diff
@@ -1,7 +1,61 @@
+console.log ("I'm ready!");
+
 // Iteration 1: Names and Input
 
+let hacker1 = 'Joao';
+console.log(`The driver's name is ${hacker1}!`);
+
+let hacker2 = 'Victoria';
+console.log(`The navigator's name is ${hacker2}!`);
+
 
 // Iteration 2: Conditionals
 
+if (hacker1.length > hacker2.length) {
+    console.log(`The driver has the longest name, it has ${hacker1.length} characters.`);
+
+  } else if (hacker1.length < hacker2.length) {
+    console.log(`It seems that the navigator has the longest name, it has ${hacker2.length} characters.`);
+    
+  } else {
+    console.log(`Wow, you both have equally long names, ${hacker1.length} characters!`);
+  }
 
 // Iteration 3: Loops
+
+//3.1 
+
```

Comment:
In this case, the lab is asking us to use loops for this iteration. 
Separating the word with spaces, can be done using a `for` loop.
Here's a possible solution you could've used:
```
let driver = '';
for (let i = 0; i < hacker1.length; i++) {
  driver = driver + hacker1[i] + ' ';
}

console.log(driver.toUpperCase());
```